src/component/variantPage/BasicInfo.tsx

```diff
@@ -235,18 +255,23 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
             key: 'tsg',
             category: 'tsg',
         });
-        // protein change
-        parsedData.push({
-            value: transcript.hgvspShort,
-            key: 'hgvsShort',
-            category: 'default',
-        });
-        // variant classification
-        parsedData.push({
-            value: transcript.variantClassification,
-            key: 'variantClassification',
-            category: getMutationTypeClassName(transcript),
-        });
+
+        // Check if variant is a VUE
```

Comment:
As far as I understand, the logic here is trying to say, check if this variant is VUE, if it's VUE, don't add protein change and variant classification, and read from VUE component instead. If it's not VUE, add default protein change and variant classification.

So better to change the comment to indicate this is the situation that "add default protein change and variant classification when it is not VUE"